scripts/spot-grid.py

```diff
@@ -35,7 +35,8 @@
 PIXEL_SIZE_SAMPLE_PLANE = 3.45e-6  # m
 DEFAULT_MAGNIFICATION = 40
 PIXEL_SIZE = PIXEL_SIZE_SAMPLE_PLANE / DEFAULT_MAGNIFICATION
-
+PITCH = 3.2e-6
```

Comment:
Maybe in future we can get this value from `voltage_adjustment.yaml` which is system specific

Reply:
Or maybe get it from the detector metadata?

Reply:
Good comment. I refactored this code and added the pitch as an input argument, similar to how the magnification is used.